services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/ScanEnhancedRequest.java

```diff
@@ -153,7 +171,8 @@ public int hashCode() {
         private Integer limit;
         private Boolean consistentRead;
         private Expression filterExpression;
-        private List<String> attributesToProject;
+        private List<String> attributesToProjectStringList;
```

Comment:
attributesToProjectStringList attribute is required to maintain the order of the older attributeToProject in cases of resets.